stdlib/sqlite3/dbapi2.pyi

```diff
@@ -113,6 +113,11 @@ if sys.version_info < (3, 8):
         def display(self, *args, **kwargs) -> None: ...
         def get(self, *args, **kwargs) -> None: ...
 
+class _AggregateProtocol(Protocol):
+    def __init__(self) -> None: ...
```

Comment:
Is `__init__()` really a necessary part of the protocol?

Reply:
whoops. Forgot the name str there.

Reply:
Actually no, no name there. But see above, will change to a callback protocol.